core/bootstrap/src/main/java/org/wildfly/swarm/bootstrap/modules/ContainerModuleFinder.java

```diff
@@ -89,6 +89,40 @@ public void buildModule(ModuleSpec.Builder builder, ModuleLoader delegateLoader)
                         null,
                         ModuleIdentifier.create("javax.enterprise.api"), false));
 
+        builder.addDependency(
```

Comment:
Rereading your comment: "in a WAR". Yes, that's something we can verify. I think it should work ootb, but I am not sure.